Utils/src/LeptonProducer.cc

```diff
@@ -347,27 +370,31 @@ bool LeptonProducer::ElectronID(const pat::Electron & electron, const reco::Vert
   double ooemoop = fabs(1.0/electron.ecalEnergy() - electron.eSuperClusterOverP()/electron.ecalEnergy());
   double d0vtx = electron.gsfTrack()->dxy(vtx.position());
   double dzvtx = electron.gsfTrack()->dz(vtx.position());
+  bool hovere_pass = false;
+  bool reqConvVeto[4] = {true, true, true, true};
 
   if (electron.isEB()) {
-    return eb_deta_cut[level] > fabs(dEtaIn)
-      && eb_dphi_cut[level] > fabs(dPhiIn)
-      && eb_ieta_cut[level] > sieie
-      && eb_hovere_cut[level] > hoe
-      && eb_d0_cut[level] > fabs(d0vtx)
-      && eb_dz_cut[level] > fabs(dzvtx)
-      && eb_ooeminusoop_cut[level] > fabs(ooemoop)
+    hovere_pass = (hovere_constant_) ? eb_hovere_cut_[level] > hoe : eb_hovere_cut_[level] + (1.12/electron.energy()) + (0.0368*rho/electron.energy()) > hoe;
```

Comment:
consider making the hovere function parameters configurable, they might change in the future

Reply:
Done